src/shared/module-util.c

```diff
@@ -3,11 +3,47 @@
 #include <errno.h>
 
 #include "module-util.h"
+#include "proc-cmdline.h"
+#include "strv.h"
+
+static int blacklist_modules(const char *p, char ***blacklist) {
```

Comment:
please rename blacklist -> denylist everywhere (apart from the cmdline parser argument)